code/controllers/configuration/entries/general.dm

```diff
@@ -61,6 +61,8 @@
 
 /datum/config_entry/flag/allow_vote_mode	// allow votes to change mode
 
+/datum/config_entry/flag/allow_vote_map	// allow votes to change map

```

Comment:
You changed the name of this entry why?

Reply:
this is just an entry to allow people to start map votes